.github/workflows/snippet_test.yml

```diff
@@ -7,7 +7,7 @@ on:
   workflow_dispatch:
 
 env:
-  POETRY_VERSION: 1.4.2
+  POETRY_VERSION: 1.8.3
```

Comment:
Why do we need version bump here?

Reply:
poetry released version `1.8.3` 6 months ago. I believe it is stable enough for our adoption: https://github.com/python-poetry/poetry/releases

Initially, I assumed the older version was causing a lint error, but the poetry version is not related to the linter.

I can revert back the poetry-related changes, if required.

Reply:
No issue at all. Just curious since poetry.lock already use 1.8.3 but I guess it doesn't update automatically in all places.
https://github.com/XRPLF/xrpl-py/blob/main/poetry.lock#L1